src/vs/workbench/contrib/notebook/browser/controller/executeActions.ts

```diff
@@ -79,20 +80,21 @@ async function runCell(editorGroupsService: IEditorGroupsService, context: INote
 		}
 	}
 
+	let earlyRevealFlag = false;
 	if (context.ui && context.cell) {
+		earlyRevealFlag = handleSmartAutoReveal(context.cell, context.notebookEditor, configurationService);
```

Comment:
`handleSmartAutoReveal` also needs `context.autoReveal` otherwise ctrl+enter on a cell will scroll the editor.